packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -172,47 +183,30 @@ module.exports = {
       // in the main CSS file.
       {
         test: /\.css$/,
-        loader: ExtractTextPlugin.extract('style', 'css?importLoaders=1!postcss')
+        loader: ExtractTextPlugin.extract({
+          fallbackLoader: 'style-loader',
+          loader: [
+            {
+              loader: 'css-loader',
+              query: {
```

Comment:
That change is pending on https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/281, which I believe has gone through.